src/Microsoft.AspNetCore.ResponseCaching/Interfaces/IResponseCacheKeyProvider.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-
-namespace Microsoft.AspNetCore.ResponseCaching
-{
-    public interface IResponseCacheKeyProvider
```

Comment:
If it's part of the future plans there's no reason to delete it, we're just making it internal because we're worried it's not done yet.